report_aeroo/tests/test_report_aeroo.py

```diff
@@ -72,7 +79,7 @@ def test_03_sample_report_pdf_with_attachment(self):
 
     def test_04_libreoffice_low_timeout(self):
         self.env['ir.config_parameter'].set_param(
-            'report_aeroo.libreoffice_timeout', '0.01')
```

Comment:
@yasmineelmrini I would prefer to keep the original parameter name please. Otherwise, we have to remember to change the parameter on every projects.